src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -8,6 +8,12 @@ set(CORECONSOLE_SOURCES
     coreconsole.cpp 
 )
 
+if(NOT WIN32)
```

Comment:
I believe the 'NOT WIN32' check isn't necessary in this file and the ones for the other Unix/OS X hosts. Looking at the CMakeLists.txt in the parent (hosts) directory, the unixcorerun, unixcoreconsole, and osxbundlerun directories are only included if it's not WIN32.